core/src/main/groovy/org/grails/gorm/graphql/entity/operations/CustomOperation.groovy

```diff
@@ -35,7 +35,7 @@ abstract class CustomOperation<T> implements Named<T>, Describable<T>, Deprecata
     DataFetcher dataFetcher
     boolean defaultListArguments = false
 
-    T dataFetcher(DataFetcher dataFetcher) {
+    T dataFetcher(DataFetcher<?> dataFetcher) {
```

Comment:
This was a bit bizarre, but when the implementing GORM entity used @GrailsCompileStatic, this would result in a static compilation error without the generic wildcard.